src/tools/clippy/tests/ui/dbg_macro.stderr

```diff
@@ -103,13 +103,12 @@ error: the `dbg!` macro is intended as a debugging tool
   --> $DIR/dbg_macro.rs:53:5
    |
 LL |     dbg!();
-   |     ^^^^^^^
+   |     ^^^^^^
    |
 help: remove the invocation before committing it to a version control system
    |
-LL -     dbg!();
-LL +     
-   |
+LL |     ();
+   |     ~~
```

Comment:
The Clippy stderr file changes look really sus. Why did they change by just moving the `lint_store` to `Session`?

Reply:
Ah, just saw the conversation above. I'd rather not merge this Clippy regression, until the fixes for this in Clippy are synced to the Rust repo.

Reply:
As @Alexandoo described [above](https://github.com/rust-lang/rust/pull/117649#issuecomment-1798543616), that test is badly written and this change just exposes the latent badness.

Reply:
What's the mechanism and timing for Clippy syncing into the Rust repo?

Reply:
The next sync is on Thursday.